synapse_data_warehouse/synapse/dynamic_tables/V2.22.0__usergroup_latest.sql

```diff
@@ -0,0 +1,23 @@
+use schema {{database_name}}.synapse; --noqa: JJ01,PRS,TMP,CP01
+
+CREATE OR REPLACE DYNAMIC TABLE USERGROUP_LATEST
+    TARGET_LAG = '1 day'
+    WAREHOUSE = compute_xsmall
+    AS
+        WITH dedup_usergroup AS (
+            SELECT
+                *,
+                "row_number"()
```

Comment:
Why did you put "row_number" in quotes?

Reply:
I borrowed this from this R__populate_team_latest.sql. I think this is not necessary though since ROW_NUMBER is default function. But @jaymedina do you know why quotation is used in this case?

Reply:
No particular reason, but yes I think we should stick to `ROW_NUMBER` to be consistent. This is from an older query that I migrated out into its own script.

Reply:
@jaymedina Thanks so much for the clarification. @philerooski do you want me to update it in another PR?

Reply:
Let's not worry about it right now since it's already been committed, next time we use this function, we can be consistent. Thanks everyone for catching this!

Reply:
Thanks, I was just curious. It only affects readability.

Reply:
Sounds great. Thanks everyone!